fixtures/framework/README.md

```diff
@@ -9,59 +9,77 @@ The following tools should already be installed
 1. docker
 2. docker-compose
 
-### Starting STRATO
 
-Start STRATO using the following environment variables
+### Obtaining an ADMIN TOKEN
+
+This project requires an `ADMIN_TOKEN` environment varibale for the app deployment to work. This token can be obtained by using the `token-getter` utility packaged in `blockapps-rest`. To execute this utility, run `sudo yarn token-getter` from the `<dir>-server` directory:
 
 ```
-OAUTH_ENABLED=true OAUTH_CLIENT_ID=<client-id> OAUTH_CLIENT_SECRET=<client-secret> OAUTH_DISCOVERY_URL=<discovery-url> OAUTH_JWT_USERNAME_PROPERTY=email HTTP_PORT=8080 NODE_HOST=localhost:8080 ./strato.sh --single
+cd <dir>/<dir>-server
+sudo yarn token-getter
 ```
 
-### Obtaining an ADMIN TOKEN
-
-This project requires an `ADMIN_TOKEN` env varibale for the deploy to work. This token can be obtained by using the `token-getter` utility packaged in `blockapps-rest`. This utility can be executed by running the following command:
-
+This command launches a small web server on port 80. 
+- In your brower, go to `http://localhost`
+- Log in with your openID credentials
+- Once logged in, the web server will display the token on a web page. 
+- Copy the "Access Token".
+- Hit `CTRL+C` to quit the `token-getter`.
+- Paste the token into the `.env` file under the `<dir>-server` folder, and label it as the environment variable `ADMIN_TOKEN`. Example:
 ```
-yarn token-getter
+ADMIN_TOKEN=eyJhbGci.....
 ```
 
-This command launches a small web server on port 8000 and open a browser window. The user can now login with the admin credentials. Once logged in, the web server will display the token on a web page. This token can be copied in pasted into a `.env` file under the `<dir>-server` folder. Alternatively, it can be supplied as a runtime environment variable by setting the `ADMIN_TOKEN` environment variable when running `yarn start`.
 
-### Executing for development
+### Executing for Development
 
 #### Start nginx
 
 ```
-cd nginx-docker
-HOST_IP=$(ipconfig getifaddr en0) docker-compose up -d
+cd <dir>/nginx-docker
+HOST_IP=$(curl ifconfig.me) docker-compose up -d
```

Comment:
this won't work properly if the host is behind the NAT. Not every machine in the network has the public IP address.
Let's use `172.17.0.1` for linux and `docker.for.mac.localhost` for mac.